classes/02class/exercises/c02-network02/ramiray/c02-network02.txt

```diff
@@ -0,0 +1,16 @@
+1.10.0.0.0/16
+
+    Subnet        Network 		    Host Range         Usage Hosts    Broadcasts       AWS Reserved(1st 4 ips and last 1 ip)
+subnet a    10.0.0.0/21		10.0.0.1 - 10.0.7.254	    2046		10.0.7.255		10.0.0.1,10.0.0.2,10.0.0.3,10.0.0.4,10.0.7.254			
+subnet b    10.0.8.0/21		10.0.8.1 - 10.0.15.254	    2046		10.0.15.255     10.0.8.1,10.0.8.2,10.0.8.3,10.0.8.4,10.0.15.254
+subnet c    10.0.16.0/20	10.0.16.1 - 10.0.31.254	    4094		10.0.31.255     10.0.16.1,10.0.16.2,10.0.16.3,10.0.16.4,10.0.31.254
+subnet d    10.0.32.0/19	10.0.32.1 - 10.0.63.254	    8190		10.0.63.255     10.0.32.1,10.0.32.2,10.0.32.3,10.0.32.4,10.0.63.254
```

Comment:
There is nothing wrong in splitting them into different netmasks, as long as they don't overlap -- it is harder to review but I think its a good way to split.

I think this looks very good, except for the "AWS Reserved" where the first IP should be one IP before. See my comment below